libs/core/src/lib/bar/directives/bar-right.directive.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { BarModule } from '../bar.module';
+
+@Component({
+    template: `
+        <div #directiveElement fd-bar-right>Bar Right Area Test</div>
+    `
+})
+class TestComponent {
+    @ViewChild('directiveElement', { static: false })
+    ref: ElementRef;
+}
+
+describe('ActionBarActionsDirective', () => {
```

Comment:
Let's take a look again on test names, there are a lot of wrong named

Reply:
Done